change/@microsoft-teams-js-db4c1ae0-ce9d-45a5-a3f1-2f9c8d825f1d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "add code comment for dialog.submit",
```

Comment:
Changefile formatting should match how we describe it in Contributing.md. Comments should be in past tense and code references should be surrounded by back ticks. Something like:

"Added code comment for `dialog.submit`"

This format matches how we want the patch notes to read and makes less work for people making the releases.